tests/tests_basic.yml

```diff
@@ -294,22 +294,17 @@
             podman_selinux_ports: []
 
         - name: Check that pods are not running
-          command: podman pod inspect {{ item[0] }} --format {{ __fmt | quote }}
+          command: podman pod exists {{ item[0] }}
```

Comment:
In theory a Pod can exist and not be running.  So we may need to have a `pod exists && pod inspect --format` but I think for the test it's enough.